src/go/rpk/pkg/cli/debug/bundle/bundle.go

```diff
@@ -143,14 +145,15 @@ func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 				out.MaybeDie(err, "unable to parse label-selector flag: %v", err)
 				bp.labelSelector = labelsMap
 			}
-			// To execute the appropriate bundle we look for
-			// kubernetes_service_* env variables to identify if we are in a
-			// k8s environment.
-			host, port := os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")
-			if len(host) == 0 || len(port) == 0 {
-				err = executeBundle(cmd.Context(), bp)
-			} else {
+
+			if isK8sEnv() {
+				fmt.Println("Detected K8s env")
```

Comment:
What do you think of moving this to debug?
```suggestion
				zap.L().Sugar().Debug("Detected K8s env")
```

Reply:
Same for Docker env

Reply:
Hmm, I'm fine with either - as it's a debug command we're running already, my thinking was that a bit more info while runing is fine, but happy to drop it down if you'd rather